tests/test_pysnooper.py

```diff
@@ -178,24 +177,26 @@ def square(self):
         prefix='ZZZ'
     )
 
-def test_file_output():
 
+def test_file_output():
     with temp_file_tools.create_temp_folder(prefix='pysnooper') as folder:
         path = folder / 'foo.log'
+
         @pysnooper.snoop(str(path))
-        def my_function(foo):
+        def my_function(_foo):
```

Comment:
At least to my IDE, the leading underscore indicates an unused variable so it doesn't show a warning about being unused.